### DISCUSSION TITLE
Add Tuya Motion Sensor ZM-35ZH-Q?

### DISCUSSION THREAD

**Discussion by tottelitott** [Koenkk/zigbee2mqtt#7345] [Devices]
Hi guys...

i bought this nice little motion sensor: https://de.aliexpress.com/item/1005002346449643.html?spm=a2g0s.9042311.0.0.27424c4dz1pdOV

But sadly it's unknown... What can i do to add this device? 

New to this ;)

Thanks in advance...

**Comment by jchurchward** [comment]
+1 I purchased two of these as well and I would love to see support for them. 

Also looks like this motion sensor came out with a new hub and other sensors. https://www.aliexpress.com/item/1005002576945995.html

**Reply by jchurchward** [reply]
Anyone got this working with Z2M? I'd love to try the couple I purchased.

**Reply by iMicknl** [reply]
@jchurchward did you already got it working? Looking to purchase them as well, they look really small on product photos.

**Reply by jchurchward** [reply]
No not in the latest release. I believe i found some people working on it recently though so could be in a dev or next main release.

**Reply by joohann** [reply]
Also waiting for it :)
